packages/wrangler/src/dialogs.tsx

```diff
@@ -156,3 +156,17 @@ export async function fromDashMessagePrompt(
 		return await confirm("Would you like to continue?");
 	}
 }
+
+export async function tailDOLogPrompt(
+	scriptName: string
+): Promise<boolean | void> {
+	logger.warn(
+		`Beginning log collection requires restarting the Durable Objects associated with ${chalk.green(
```

Comment:
I wonder if we should keep the tail terminology rather than log collection, which is not used anywhere else?

```suggestion
		`Tailing requires restarting the Durable Objects associated with ${chalk.green(
```

Reply:
This was suggested by the design team because we're adding a similar warning to the dashboard—but I agree sticking to the `tail` terminology might be better for Wrangler